script-test/playbackstrategies/msestrategytest.js

```diff
@@ -1,491 +1,579 @@
+          });
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#r=0', true);
-        });
+          it('should playback from relative live start time for video simulcast', function () {
+            setUpMSE(0, WindowTypes.SLIDING, MediaKinds.VIDEO);
 
-        it('should playback from derived start time for webcast', function () {
-          setUpMSE(0, WindowTypes.GROWING, MediaKinds.VIDEO, 100000, 200000);
+            mockDashInstance.getSource.and.returnValue('src');
 
-          mockDashInstance.getSource.and.returnValue('src');
+            // Actual live point requested
+            mseStrategy.load('src', null, 0);
 
-          mseStrategy.load('src', null, 0.1);
+            expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#r=-1', true);
+          });
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#t=101', true);
-        });
+          it('should playback from relative live start time for audio simulcast', function () {
+            setUpMSE(0, WindowTypes.SLIDING, MediaKinds.AUDIO);
 
-        it('should playback from the provided time of a webcast', function () {
-          setUpMSE(0, WindowTypes.GROWING, MediaKinds.VIDEO, 100000, 200000);
+            mockDashInstance.getSource.and.returnValue('src');
 
-          mockDashInstance.getSource.and.returnValue('src');
+            // Actual live point requested
+            mseStrategy.load('src', null, 0);
 
-          mseStrategy.load('src', null, 60);
+            expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#r=-1', true);
+          });
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#t=160', true);
-        });
+          it('should playback from relative start time for video simulcast', function () {
+            setUpMSE(0, WindowTypes.SLIDING, MediaKinds.VIDEO);
 
-        it('should playback from the live point of a webcast', function () {
-          setUpMSE(0, WindowTypes.GROWING, MediaKinds.VIDEO, 100000, 200000);
+            mockDashInstance.getSource.and.returnValue('src');
 
-          mockDashInstance.getSource.and.returnValue('src');
+            // Somewhat live, playbackUtils forces 0.1 for TAL related reasons!
+            mseStrategy.load('src', null, 0.1);
 
-          mseStrategy.load('src', null, undefined);
+            expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#r=0', true);
+          });
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src', true);
-        });
-      });
+          it('should playback from relative start time for audio simulcast', function () {
+            setUpMSE(0, WindowTypes.SLIDING, MediaKinds.AUDIO);
 
-      describe('getSeekableRange()', function () {
-        it('returns the correct start and end time', function () {
```

Comment:
As per my above comments re: duration, I don't think it's necessary to test audio and video as the code path is the same.